tests/integration.rs

```diff
@@ -425,6 +425,8 @@ fn test_tree() {
     cmd()
         .arg(tmp.path())
         .arg("--tree")
+        .arg("--color")
+        .arg("never")
```

Comment:
@meain could you have a look at this? The tests failed because of the colours so i added the option but apparently these tests were absolutely fine before. If you could perhaps try running the tests without the changes here and see if they work or not? It may be the case that I've changed something inadvertently but I can't figure it out from reading the code.

Reply:
I removed these and tried out in my local. It seems to work even without `--color never` in my local. It is possible that this was introduced because of the change in the lib that gives us term width. If `lsd` detects that it is not an interactive terminal, we disable colors automatically. This is done by checking the term width.

I was actually able to run this without these and it seems to be fine. (https://github.com/meain/lsd/actions/runs/685425620). Do you remember which platform it was failing for?

Reply:
Strange, it's only failing locally for me. Even when reverted to `terminal_size`

Reply:
Solved it! lsd was reading my config file which had `--color always`. Check my recent push.

Reply:
I think we should probably add `--ignore-config` in there.